Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tracing-tree"
-version = "0.2.5"
+version = "0.2.6"
```

Comment:
actually, hold on—this needs to be bumped to 0.3.0 because we removed features in `tracing-log` and I didn't specify _which_ features this library uses. i'll push to your branch, one sec.

Reply:
Yeah, I was actually wondering about that.

Reply:
I won't do the bump in this PR, but I'll do the bump in a subsequent PR. ease of reading history.